gdsc-web-backend/Database/AppDbContext.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace gdsc_web_backend.Database
+{
+    public class AppDbContext :DbContext
```

Comment:
You've forgotten a space after colon =)))

Reply:
Am rezolvat :)